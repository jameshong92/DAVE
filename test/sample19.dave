// test table definition2
int main() {
  rec r1 = new rec(age: 22, name:"Yang", GPA:3.50); 
  rec r2 = new rec(age:23, name:"James", GPA:3.88);
  rec r3 = new rec(age:24, name:"Min Woo", GPA:3.80);
  rec[] r4 = [r1, r2, r3];
  tbl test = new tbl(r4);
	print(test);
	print(test.row_length);
	print(test.col_length);
}