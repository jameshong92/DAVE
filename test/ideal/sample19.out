age	name	GPA	
22	Yang	3.5	
23	James	3.88	
24	Min Woo	3.8	

3
3
